internal/satellite/telemetry/telemetry.go

```diff
@@ -0,0 +1,53 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package telemetry
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Registerer is the global metrics center for collecting the telemetry data in core modules or plugins.
+var (
+	registry   *prometheus.Registry
+	Registerer prometheus.Registerer // The register is for adding metrics to the registry.
+	Gatherer   prometheus.Gatherer   // The gatherer is for fetching metrics from the registry.
+)
+
+// Config defines the common telemetry labels.
+type Config struct {
+	Service   string `mapstructure:"service"`   // The service name.
+	Namespace string `mapstructure:"namespace"` // The kubernetes namespace.
+	Pod       string `mapstructure:"pod"`       // The kubernetes pod name.
```

Comment:
IIRC, we should not introduce Kubernetes notions into satellite since it's designed to deployed on VM as well. 

How about using more general and SkyWalking-style concepts, for example, `Cluster`, `Service` and `Instance`

Reply:
no problem